### DISCUSSION TITLE
Spawn Potential Scripts

### DISCUSSION THREAD

**Discussion by BisUmTo** [gnembon/scarpet#192] [Ideas]
Should Spawn Potential Scripts be included in this repository?
( Video: https://youtu.be/ZlefdidnABI )

**Comment by altrisi** [comment]
Problem is, since those are just in the description of the video and don't explicitly state a license, we don't 100% know if they can be shared under CC0-1.0 license. Since it's from gnembon he is probably ok with it, but it would need asking just in case.

**Reply by BisUmTo** [reply]
We must wait @gnembon permission... 🤔
